packages/battery_plus/example/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dev_dependencies:
     sdk: flutter
   integration_test_tizen:
     path: ../../integration_test/
-  flutter_lints: ^1.0.4
+  flutter_lints: ^2.0.1
```

Comment:
This line could be removed. Please update the dependency in `battery_plus/pubspec.yaml` instead.

Reply:
I updated code. thanks